install-k

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -e
 
-K_VERSION=5.4.13
```

Comment:
These changes should likely be undonel. It's likely it's from checking out the master branch locally when starting your own branch. Can be fixed with a rebase onto develop.

Reply:
I will wait for the dust to settle. When I based on develop on a different PR on Friday, none of the CI runs were getting triggered, but that might have been unrelated...